modules/layers/src/text-layer/font-atlas.js

```diff
@@ -22,6 +19,18 @@ function getDefaultCharacterSet() {
 }
 
 export const DEFAULT_CHAR_SET = getDefaultCharacterSet();
+export const DEFAULT_FONT_FAMILY = 'Monaco, monospace';
+
+export const DEFAULT_FONT_SETTINGS = {
+  characterSet: DEFAULT_CHAR_SET,
+  fontFamily: DEFAULT_FONT_FAMILY,
```

Comment:
Don't put this field in `DEFAULT_FONT_SETTINGS` if it's not part of `fontSettings`.